addons/website_slides/views/website_slides_templates_course.xml

```diff
@@ -566,12 +566,14 @@
                 <div class="d-flex align-items-center justify-content-between border-bottom pb-2 mb-3" t-if="category['name'] != 'Uncategorized'">
                     <t t-set="is_empty_editable" t-value="not category['slides'] and channel.can_publish"/>
                     <h5 t-attf-class="m-0 #{'text-muted' if is_empty_editable else ''}"><t t-esc="category['name']"/></h5>
-                    <a t-if="category['id'] and not is_empty_editable" t-att-href="'/slides/%s/category/%s' % (slug(channel), category['slug_name'])">View all</a>
-                    <a t-elif="not is_empty_editable" t-att-href="'/slides/%s?uncategorized=1' % (slug(channel))">View all</a>
+                    <t t-if="category['total_slides'] > len(category['slides'])">
+                        <a t-if="category['id'] and not is_empty_editable" t-att-href="'/slides/%s/category/%s' % (slug(channel), category['slug_name'])">View all</a>
```

Comment:
Actually it was an fp request to always display the view all to filter on a category.